guides/15_Background_Filter_Video_Processor.md

```diff
@@ -1,46 +1,46 @@
-# Integrating background blur into your Amazon Chime SDK for JavaScript application
+# Integrating background filters into your Amazon Chime SDK for JavaScript application
 
-## What is background blur?
+## What is a background filter?
 
-The background blur API allows builders to enable background blur on a video stream. To add a background blur to a video stream the builder needs to create a `VideoFrameProcessor` using `BackgroundBlurVideoFrameProcessor` and then insert that processor into a `VideoTransformDevice`. The video frame processor uses a TensorFlow Lite (TFLite) machine learning (ML) model along with JavaScript Web Workers and WebAssembly (WASM) to apply blur to the background of each frame in the video stream. These assets are downloaded at runtime when the video frame processor is created and not provided in the source directly.
+The background filter API allows builders to enable background filtering on a video stream. You can select from background blur filtering or replacement filter processors. To add a background filter to a video stream the builder needs to create a `VideoFrameProcessor` using `BackgroundBlurVideoFrameProcessor` or `BackgroundReplacementVideoFrameProcessor` and then insert that processor into a `VideoTransformDevice`. The video frame processor uses a TensorFlow Lite (TFLite) machine learning (ML) model along with JavaScript Web Workers and WebAssembly (WASM) to apply the filter to the background of each frame in the video stream. These assets are downloaded at runtime when the video frame processor is created and not provided in the source directly.
```

Comment:
>The video frame processor uses a TensorFlow Lite (TFLite) machine learning (ML) model
nit: This should be background filter right? video frame processor is just a generic class.

Reply:
fixed